src/coreComponents/physicsSolvers/wavePropagation/AcousticTTIFields.hpp

```diff
@@ -0,0 +1,257 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+
+/**
+ * @file AcousticVTIFields.hpp
+ */
+
+#ifndef GEOS_PHYSICSSOLVERS_WAVEPROPAGATION_ACOUSTICTTIFIELDS_HPP_
+#define GEOS_PHYSICSSOLVERS_WAVEPROPAGATION_ACOUSTICTTIFIELDS_HPP_
+
+#include "common/DataLayouts.hpp"
+#include "mesh/MeshFields.hpp"
+
+
+namespace geos
+{
+
+namespace fields
+{
+
+namespace acousticttifields
+{
```

Comment:
Just for the record, you can now write this as
```suggestion
namespace geos::fields::acousticttifields
{
```